c2corg_ui/static/js/viewdetails.js

```diff
@@ -16,9 +16,9 @@ app.viewDetailsDirective = function() {
     link: function(scope, el, attrs, ctrl) {
 
       function initGalleries() {
-        ctrl.initSlickGallery_();
```

Comment:
Do we still use the slick carousel library?
If not it would make sense to clean it completely out: https://github.com/c2corg/v6_ui/search?utf8=%E2%9C%93&q=slick&type=

Reply:
fixed